app/lib/features/activities/providers/notifiers/notifications_list_notifier.dart

```diff
@@ -76,17 +71,17 @@ class NotificationsListNotifier extends StateNotifier<NotificationListState>
     _listener = client.notificationsStream();
     if (_listener != null) {
       _poller = _listener!.listen((ev) {
-        debugPrint(
-          ' --- - - ----------------- new notification received',
-        );
+        // debugPrint(
+        //   ' --- - - ----------------- new notification received',
+        // );
 
-        final provider = ref.watch(featuresProvider);
-        if (provider.isActive(LabsFeature.showNotifications)) {
-          if (!ev.read()) {
-            final brief = extractBrief(ev);
-            notify(brief);
-          }
-        }
+        // final provider = ref.watch(featuresProvider);
```

Comment:
remove commented

Reply:
this was actually from a previous PR that is now not part of this PR anymore as it is already in `main`... If you still want to discuss that, let's address it separately.